src/ringct/rctSigs.cpp

```diff
@@ -973,7 +976,7 @@ namespace rct {
           }
         }
         else {
-          const key message = get_pre_mlsag_hash(rv);
+          const key message = get_pre_mlsag_hash(rv, hw::get_device("default"));
```

Comment:
Why "default" ?

Reply:
Verification does not imply any secret, so the real hw device is not required here. So no need to pass it to the function, the default is sufficient.